crates/milli/src/index.rs

```diff
@@ -113,7 +113,7 @@ pub mod db_name {
     pub const FIELD_ID_DOCID_FACET_F64S: &str = "field-id-docid-facet-f64s";
     pub const FIELD_ID_DOCID_FACET_STRINGS: &str = "field-id-docid-facet-strings";
     pub const VECTOR_EMBEDDER_CATEGORY_ID: &str = "vector-embedder-category-id";
-    pub const VECTOR_ARROY: &str = "vector-arroy";
+    pub const VECTOR_HANNOY: &str = "vector-hannoy";
```

Comment:
```suggestion
    pub const VECTOR_STORE: &str = "vector-arroy";
```

We will need to use the same database as we plan to keep the same format for the raw vectors and do the dumpless upgrade by "simply" deleting all the arroy tree nodes and ask Hannoy to generate the links by using the raw vectors. cc @nnethercott